### ISSUE TITLE
[BUG] accessibility should be improved

### ISSUE THREAD

**Issue by amgando** [near/docs#124]

**Comment by mattlockyer** [comment]
Is this still required? If so what needs improvement.
Audience accessibility, e.g. different skill levels?
HTML accessibility?

**Comment by amgando** [comment]
let's close and let it be reopened.  was just using this to capture a comment at one point
